<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE properties SYSTEM "cpf.dtd">
<properties>
<simple name="Import" type="string"><value>maestro</value></simple>

<struct name="OpenRaveIn" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1000</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>or_in</value></simple>
</struct>
<struct name="OpenRaveOut" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1000</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>or_out</value></simple>
</struct>
<struct name="HUBOCommand" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1000</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>Hubo/HuboCommand</value></simple>
</struct>
<struct name="ACHCommand" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1000</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>Hubo/AchCommand</value></simple>
</struct>
<struct name="HuboState" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1000</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>Hubo/HuboState</value></simple>
</struct>
<struct name="CANUp" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1000</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>can_up</value></simple>
</struct>


<struct name="RobotControl" type="RobotControl">
  <struct name="Activity" type="PeriodicActivity">
    <simple name="Period" type="double"><value>0.01</value></simple>
    <simple name="Priority" type="short"><value>0</value></simple>
    <simple name="Scheduler" type="string"><value>ORO_SCHED_RT</value></simple>
  </struct>
  <simple name="AutoConf" type="boolean"><value>1</value></simple>
  <simple name="AutoStart" type="boolean"><value>1</value></simple>
  <struct name="Ports" type="PropertyBag">
    <simple name="or_in" type="string"><value> OpenRaveIn </value></simple>
    <simple name="or_out" type="string"><value> OpenRaveOut </value></simple>
    <simple name="Hubo/HuboCommand" type="string"><value> HUBOCommand </value></simple>
    <simple name="Hubo/AchCommand" type="string"><value> ACHCommand </value></simple>
    <simple name="Hubo/HuboState" type="string"><value> HuboState </value></simple>
    <simple name="can_up" type="string"><value> CANUp </value></simple>
  </struct>
  <simple name="Service" type="string"><value>rosparam</value></simple>
</struct>


</properties>
